client/ayon_resolve/plugins/create/create_shot_clip.py

```diff
@@ -170,6 +170,14 @@ class ResolveShotInstanceCreator(_ResolveInstanceClipCreator):
 
     def get_instance_attr_defs(self):
         instance_attributes = CLIP_ATTR_DEFS
+        instance_attributes.append(
+            BoolDef(
+                "sourceResolution",
```

Comment:
`sourceResolution` sounds like it holds the value of resolution instead of bolean.
```suggestion
                "use_source_resolution",
```

or (not sure why camel case is used?)
```suggestion
                "useSourceResolution",
```

Reply:
I agree `sourceResolution` sounds quite confusing.